models/engine/file_storage.py

```diff
@@ -55,7 +70,7 @@ def reload(self):
                 jo = json.load(f)
             for key in jo:
                 self.__objects[key] = classes[jo[key]["__class__"]](**jo[key])
-        except:
```

Comment:
You have an empty except clause, which will cause an error. Please remove it. 
Otherwise, everything looks great.